bin/next_downtime_id

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -o errexit
+set -o pipefail
+cd "$(dirname "$0")"
+datadir=$(git rev-parse --show-toplevel)/topology
+lastid=$(grep -Eh '^[[:space:]]+ID:[[:space:]]*[[:digit:]]+' "$datadir"/*/*/*_downtime.yaml | cut -d: -f2 | awk '{print $1}' | sort -n | tail -n 1)
```

Comment:
147 characters wide in a shell script kinda grinds my gears ...

Do you really like it all on a single line?

Also, I'm a bit skeptical about the named character classes (`[:space:]` and `[:digit:]`), since they take the locale into account.  Does this mean we want to allow weird unicode space and digit characters?

Anyway if you like it how you have it, that's fine.  Otherwise I might suggest something like

```
lastid=$( grep -Eh '^ +ID: *[0-9]+' "$datadir"/*/*/*_downtime.yaml |
          awk '{print $2}' | sort -n | tail -n 1 )
```

Reply:
Yes, that looks better.